### ISSUE TITLE
GLaDOS's eye is not orange at beginning of her bootup

### ISSUE THREAD

**Issue by alebahn** [ValveSoftware/portal2#27]
When GLaDOS first boots, her orange pupil did not show. After the assembly animation and the player is picked up, the eye is orange.

**Comment by RoninDusette** [comment]
Nice. You are a true gamer. haha :) Trivial, but details do count. Its not my project, but it for sure helps to give more info. Like, your system specs and distro. Without that, its impossible to tell what the issue might be. Terminal output helps, too. Just FYI.

**Comment by alebahn** [comment]
I didn't have the console open at the time.
I'm running Ubuntu 12.04 3.2.0-58-generic x86_64
My graphics: NVIDIA Corporation GF104 [GeForce GTX 460 SE](rev a1)
driver: nvidia 331-updates

**Comment by RoninDusette** [comment]
Word. Ill bet its Unity. Is it possible to try another Desktop Environment? I like XFCE for testing stuff like this, as its super lightweight. I would do that, reboot, and when you log in, you should be able to click a drop down menu before you actually click LOGIN that will let you select XFCE for your DE for that session. 

You should be able to just install it with:

sudo apt-get install xfce4

or

sudo apt-get install xubuntu-desktop 

xubuntu-desktop is a meta package and will pull all apps for XFCE. I would try that and see if it makes a difference in it. Unity is funky.

**Comment by Shished** [comment]
Can [confirm](http://cloud-3.steampowered.com/ugc/3316078062237193333/4060D2E5B3658814738467A9545F04223F9C2746/)

My system is openSUSE 13.1 x86_64 (gnome-shell) with linux 3.13.5 with nvidia GTX 660 and 334.16 driver.

**Comment by danhedron** [comment]
@RoninDusette What on earth makes you think this is anything to do with the Desktop Environment?

I also seem to be experiencing this issue, KDE 4.11, Kubuntu 13.10.

**Comment by RoninDusette** [comment]
Its called troubleshooting. I help admin playonlinux, and troubleshoot
graphics issues daily. Obviously, since you can confirm it is the same on
KDE, then that is not the case. Again, its called troubleshooting. I won't
help out anymore. What do I know, right? Hope you figure it out.

**Comment by dimgel** [comment]
Confirmed. Gentoo/amd64, 
01:00.0 VGA compatible controller: NVIDIA Corporation GK107M [GeForce GT 650M](rev a1)
UPD: Even after she grabs me, her eye is small (just pupil) and turns off when she pauses speeking.

**Comment by dos1** [comment]
Same as above on my Debian @ amd64 with Intel HD Graphics 4000.

**Comment by Arbitrary007** [comment]
I experienced this as well, though the later chamber where you meet back up with GlaDOS after disabling the neurotoxin and turrets has her glowing eye looking as it should.

**Comment by Arbitrary007** [comment]
Playing through the boot-up stage does still look like she doesn't have an eye at first, but when I moved around in the little pit I came up in, there are some spots where I see her eye and others I do not. Her eye also looks more like a dot of light rather than a yellow lens like she has when you finally confront her. I wonder if the light I see is the backlight for her eye and the eye just isn't textured.

**Comment by ghost** [comment]
Confirmed here. System information: https://gist.github.com/CharlesAtum/2e5f6b8deaf85ba6c472
Running with optirun -b primus on a Nvidia Geforce GT630M

Also, GLaDOS eye will be blinking orange at the first moments of the boot-up.

Here are some screenshots I took:
http://imgur.com/a/VhRjL

**Comment by MichaelAquilina** [comment]
I noticed the same problem today. Kubuntu 13.10 64 bit GT555m

**Comment by Soapsuds** [comment]
Just saw this problem as well. Archlinux Nvidia 560ti 331.49.

**Comment by j5lx** [comment]
Same issue here.

```
Processor Information:
    Vendor:  GenuineIntel
    CPU Family:  0x6
    CPU Model:  0x3a
    CPU Stepping:  0x9
    CPU Type:  0x0
    Speed:  2400 Mhz
    2 logical processors
    2 physical processors
    HyperThreading:  Unsupported
    FCMOV:  Supported
    SSE2:  Supported
    SSE3:  Supported
    SSSE3:  Supported
    SSE4a:  Unsupported
    SSE41:  Supported
    SSE42:  Supported

Network Information:
    Network Speed:  

Operating System Version:
    "Arch Linux" (64 bit)
    Kernel Name:  Linux
    Kernel Version:  3.13.4-1-ARCH
    X Server Vendor:  The X.Org Foundation
    X Server Release:  11500000
    X Window Manager:  Enlightenment
    Steam Runtime Version:  steam-runtime-release_2014-02-05

Video Card:
    Driver:  Intel Open Source Technology Center Mesa DRI Intel(R) Ivybridge Mobile x86/MMX/SSE2

    Driver Version:  3.0 Mesa 10.0.3
    OpenGL Version: 3.0
    Desktop Color Depth: 24 bits per pixel
    Monitor Refresh Rate: 60 Hz
    VendorID:  0x8086
    DeviceID:  0x156
    Number of Monitors:  1
    Number of Logical Video Cards:  1
    Primary Display Resolution:  1366 x 768
    Desktop Resolution: 1366 x 768
    Primary Display Size: 13,54" x 7,60"  (15,51" diag)
                                            34,4cm x 19,3cm  (39,4cm diag)
    Primary VRAM Not Detected

Sound card:
    Audio device: Intel PantherPoint HDMI

Memory:
    RAM:  7883 Mb

Miscellaneous:
    UI Language:  English
    LANG:  de_DE.UTF-8
    Microphone:  Not set
    Total Hard Disk Space Available:  930193 Mb
    Largest Free Hard Disk Block:  402100 Mb

Installed software:

Recent Failure Reports:
```

**Comment by stevethepocket** [comment]
Confirmed; I couldn't remember if it was supposed to be that way or not.

Radeon HD 4850, Kubuntu 14.04 Alpha 2, whatever version of the open source Radeon driver is included with that.

**Comment by TapGhoul** [comment]
Confirmed, https://github.com/ValveSoftware/portal2/issues/117 for screenshots.

**Comment by WillMcDade** [comment]
Confirmed: Arch Linux KDE. I had recorded but the recording had lost about 10 minutes of itself? So only the part of getting to the testing track recorded where Wheatly controls the cabin.

**Comment by roman-bronis** [comment]
Confirmed: Arch 64bit, Fluxbox, Catalyst 14.1 beta 1.3, AMD Radeon HD 6870

map sp_a1_wakeup

**Comment by Arbitrary007** [comment]
Tested today after the recent patch. The orange eye does have better visibility, but I still feel something is amiss. The eye during the bootup sequence still looks different than when you meet back up with her later with Wheatley.

**Comment by mariodonofrio** [comment]
Yes, well spotted, the 'better' eye was a side effect of the trilinear fix we made (for #101), we're yet to get to a 100% fix for this.

**Comment by Arbitrary007** [comment]
I try to keep an eye on any of the particular bugs I have run across and ensure they work on my system after every patch, whether it says it fixed it or not. (An accidental fix is still a fix.)

**Comment by Arbitrary007** [comment]
Ok, after watching my PS3 run through this part, the eye in the Linux version looks like the PS3 version. Maybe the lens isn't supposed to be there? Or it is and it's not a Linux specific bug.

**Comment by dos1** [comment]
Maybe it's just specific to OpenGL backend. The lens is for sure present when playing on Windows or with Wine.

**Comment by simplyhexagon** [comment]
It seems as this issue hasn't been touched in a while
In 2021, still experiencing this _bug_. Pop!_OS 20.04,
everything possible updated to latest version.

**Comment by misyltoad** [comment]
Looks the same as Windows to me on both the OpenGL and Vulkan backends.

I assume you are referring to the little sprite on her eye?

**Comment by misyltoad** [comment]
(however, if you're referring why why glados does not change skin to the one used in a2_core, there is no logic in the level to do that)

**Comment by simplyhexagon** [comment]
The orange background is missing and her eye is basically empty from a distance.
If you get closer, only then can you see a little light emitting object

**Comment by misyltoad** [comment]
I don't think that's a bug, that skin was never used in wakeup

**Comment by Soapsuds** [comment]
I can no longer reproduce this issue. AFAIK my system is producing the right result. Compare playback of the Xbox 360 version  
https://youtu.be/TKEq1bGGNjI?t=1175
And the scene on my system
https://youtu.be/AerKF--9pAA?t=83

I'm using a Vega56, Archlinux, and the recording was done on the opengl backend. 

Since my last post I've moved over to Mesa from Nvidia, but also everything else about my system and software is probably different too 😅
